daemon-tests/src/lib.rs

```diff
@@ -1122,15 +1122,38 @@ pub async fn mock_oracle_announcements(
         .await;
 }
 
-pub async fn mock_quotes(maker: &mut Maker, taker: &mut Taker) {
+pub async fn mock_quotes(maker: &mut Maker, taker: &mut Taker, contract_symbol: ContractSymbol) {
     taker.mocks.mock_latest_quotes().await;
     maker.mocks.mock_latest_quotes().await;
     let mut quote_receiver = taker.quote_feed().clone();
 
-    let non_empty_quote =
-        |quotes: projection::LatestQuotes| quotes.get(&ContractSymbol::BtcUsd).copied();
+    let non_empty_quote = |quotes: projection::LatestQuotes| quotes.get(&contract_symbol).copied();
 
     next_with(&mut quote_receiver, non_empty_quote)
         .await
         .unwrap(); // if quote is available on feed, it propagated through the system
 }
+
+pub fn expected_taker_liquidation_price(symbol: ContractSymbol, taker_position: Position) -> Price {
+    let expected_liquidation_price = match (symbol, taker_position) {
+        // inverse payout curve
+        (ContractSymbol::BtcUsd, Position::Long) => dec!(33333.333333333333333333333333),
+        (ContractSymbol::BtcUsd, Position::Short) => dec!(100_000),
+        // quanto linear payout curve
+        (ContractSymbol::EthUsd, Position::Long) => dec!(750),
+        (ContractSymbol::EthUsd, Position::Short) => dec!(2_250),
+    };
+    Price::new(expected_liquidation_price).unwrap()
+}
+
+pub fn expected_maker_liquidation_price(symbol: ContractSymbol, maker_position: Position) -> Price {
+    let expected_liquidation_price = match (symbol, maker_position) {
+        // inverse payout curve
+        (ContractSymbol::BtcUsd, Position::Long) => dec!(25_000),
+        (ContractSymbol::BtcUsd, Position::Short) => dec!(21_000_000), // INFINITE
+        // quanto linear payout curve
+        (ContractSymbol::EthUsd, Position::Long) => dec!(1),
+        (ContractSymbol::EthUsd, Position::Short) => dec!(3_000),
+    };
+    Price::new(expected_liquidation_price).unwrap()
+}
```

Comment:
🤔 It's quite unexpected to find these in a common place for all tests. These are functions where we return predefined expected values that will only ever apply if tests have been set up in a very particular way.

As much as I can understand why you'd want to reuse this, I think these should just be hard-coded in the individual tests.